paddle/string/piece.h

```diff
@@ -39,8 +39,8 @@ class Piece {
   // size_ is 0.
   Piece();
   Piece(const char* d, size_t n);
-  Piece(const char* d);
-  Piece(const std::string& s);
+  Piece(const char* d);         // NOLINT
+  Piece(const std::string& s);  // NOLINT
```

Comment:
These shouldn't be marked NOLINT but should be fixed by adding the `explicit`  keyword. Let me fix them.